library/src/main/res/values/attrs.xml

```diff
@@ -13,6 +13,17 @@
         <attr name="bpDialogBackground" format="reference"/>
     </declare-styleable>
 
+    <declare-styleable name="BetterPickersTimePickerDialog">
```

Comment:
Hi @nkarasch could you just rename this stylable as BetterPickersRadialTimePickerDialog in order to avoid confusion with the other timePicker.

Thx